api/api/configuration.py

```diff
@@ -305,6 +310,11 @@ def replace_bools(conf: dict):
     else:
         # If any value is missing from user's configuration, add the default one:
         dict_to_lowercase(configuration)
+
+        # Check if cache is enabled
+        if configuration.get('cache', {}).get('enabled', {}):
+            logger.warning(CACHE_DEPRECATED_MESSAGE.format(release="4.8.0"))
```

Comment:
The logger is only used under a specific condition, I would instantiate it here instead:
```suggestion
        # Check if cache is enabled
        if configuration.get('cache', {}).get('enabled', {}):
            logger = logging.getLogger('wazuh-api')
            logger.warning(CACHE_DEPRECATED_MESSAGE.format(release="4.8.0"))
```

Reply:
Changed in commit bf7a24717e541a34035d81c8a1124eba363c2611.